tokenizers/src/models/bpe/trainer.rs

```diff
@@ -43,6 +43,7 @@ struct Config {
     initial_alphabet: HashSet<char>,
     continuing_subword_prefix: Option<String>,
     end_of_word_suffix: Option<String>,
+    unk_token: Option<String>,
```

Comment:
I think it doesn't make sense to add the `unk_token` to the trainer. I thought about it too, but since it will never get used, this would just be misleading for the user and actually requires more work on their part. I'm fixing this with #519.